### ISSUE TITLE
[content] Weird story point in Sheragi Archaeology: The Box 2

### ISSUE THREAD

**Issue by ph2000bis** [endless-sky/endless-sky#8188]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The box is descibed as "a weathered, red metal cube, around half a meter on each side"
Later in the mission, it is said :
"The two of you put the cube in an inconspicuous backpack and carry it back to your ship"
My mind immediately asked, how can a backpack that can fit a cube that can fit a cube that size be inconspicuous ? That would make it 125L, at least 35% bigger than the biggest food delivery bag I could find : https://m.media-amazon.com/images/I/715LoDa7pHL._SL1344_.jpg
Also, such a massive thing would probably be much heavier than this mission's text suggests.

### Steps to Reproduce

1. Do the Sheragi archeology missions series to The Box 2

### Expected Behavior

Maybe shrink the box if it's coherent with the other missions (haven't played them yet), maybe a quarter of a meter is workable ? Or otherwise, get it out in a food container ?

### Operating System

Windows 11

### Game Source

Steam

### Game Version

0.9.16.1

**Comment by Naf5000** [comment]
I did have the same thought when I did that mission. For those who have trouble visualizing metric, half a meter is just under twenty inches, with a half-meter cube having a volume of 33 gallons. There are plenty of backpacks with sufficient volume, but they're both the wrong shape to transport a cube that size and also absolutely massive and highly conspicuous anywhere that isn't a several-day hike from anything.
